pkg/iostreams/iostreams.go

```diff
@@ -406,13 +406,14 @@ func System() *IOStreams {
 		term:         &terminal,
 	}
 
-	stdoutIsTTY := io.IsStdoutTTY()
+	io.stdoutIsTTY = io.IsStdoutTTY()
+	io.stderrIsTTY = io.IsStderrTTY()
```

Comment:
Setting these struct fields has no effect without also setting the override field. Let's just use local variables instead.